nUpdate.Test/UseDynamicUriTest.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
```

Comment:
When you changed the implementation, it should be clear that you should run the unit tests again to see, if there are any changes necessary.

Reply:
I ran it, and the changed implementation passed the test.